docs/release-notes/platform_10-8-0_releasenotes.md

```diff
@@ -0,0 +1,59 @@
+---
+id: v10.8.0
+title: Version 10.8.0 Release Notes
+sidebar_label: v10.8.0
+image: "img/release-note-link-preview.png"
+---
+
+_April 13, 2023_
+
+## Console
+
+### New Features
+
+#### Management of Providers trough the Console
+
+Starting from today it will be possible to manage the creation, configuration and edit of the providers used by the Console (GitProvider, Secret Manager and CI/CD Tool) directly using the respective dedicated section in the "Company Overview" area.
+
+Previously this was possible only through the use of specific APIs exposed on the API Portal, while now using a dedicated front-end on the console it will be possible to do it in a guided, simple and intuitive way.
+
+![Providers Section](./img/10.8-providers-section.png)
+
+To learn more about the providers supported by the Console and for a detailed explanation of the Section, [go to the official documentation](/development_suite/set-up-infrastructure/providers-management.md) page for more information.
```

Comment:
```suggestion
Starting from today it will be possible to manage the creation, configuration, and editing of the providers used by Mia-Platform Console (GitProvider, Secret Manager and CI/CD Tools) within a brand new dedicated section - named "Providers" - inside the "Company Overview" area. 

Previously, this was only possible through the use of specific APIs exposed on the API Portal, while now you will be able to do so directly on the Console, in a guided, simple and intuitive way.

![Providers Section](./img/10.8-providers-section.png)

To learn more about the providers supported by the Console and for a detailed explanation of the "Providers" section, [go to the official documentation](/development_suite/set-up-infrastructure/providers-management.md).
```